routes/fractie.ts

```diff
@@ -0,0 +1,45 @@
+import Router from 'express-promise-router';
+
+import { Request, Response } from 'express';
+
+import { STATUS_CODE } from '../util/constants';
+import { fractieUsecase } from '../controllers/fractie';
+import { mandatarisUsecase } from '../controllers/mandataris';
+
+export const fractiesRouter = Router();
+
+fractiesRouter.get(
+  '/:bestuursperiodeId/bestuursperiode',
+  async (req: Request, res: Response) => {
+    const id = req.params.bestuursperiodeId;
+
+    try {
+      const fractieIds = await fractieUsecase.forBestuursperiode(id);
```

Comment:
Be aware im using update & querySUDO, was having alot of issues with authorization.. How could I add a check in the router to see if the inital call is authorized?